batch/kueue-cohort/spot-cq.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_batch_kueue_cohort_spot_cq]
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: spot-cq
+spec:
+  cohort: all
+  namespaceSelector: {}
+  resourceGroups:
+  - resources: ["cpu", "memory"]
```

Comment:
```suggestion
  - coveredResources: ["cpu", "memory"]
```

Reply:
@alculquicondor is the [Kueue API Improvements doc](https://docs.google.com/document/d/1Uu4hfGxux4Wh_laqZMLxXdEVdty06Sb2DwB035hj700/edit?resourcekey=0-b7mU7mGPCkEfhjyYDsXOBg) up to date? I don't see a mention to coveredResources there

Reply:
forgot to update. Fixed now.